code/obj/item.dm

```diff
@@ -483,6 +487,16 @@
 		src.visible_message("<span class='alert'>[src] catches on fire!</span>")
 		src.burning = 1
 		src.firesource = TRUE
+		if (istype(src, /obj/item/plant))
+			if (!ON_COOLDOWN(global, "hotbox_adminlog", 30 SECONDS)) //blocks every instance of the plant logging
+				for (var/atom/movable/A in get_turf(src))
+					if (istype(A, /obj/item/plant))
+						hotbox_count = hotbox_count + 1
+						if (hotbox_count == 5) //number is up for debate, 5 seemed like a good starting place
+							if (hotbox_item)
+								message_admins("([src]) was set on fire on the same turf as multiple other plants at [log_loc(src)] by item ([hotbox_item]). Last touched by: [hotbox_item.fingerprintslast ? "[hotbox_item.fingerprintslast]" : "*null*"].")
+							else
+								message_admins("([src]) was set on fire on the same turf as multiple other plants at [log_loc(src)].")
```

Comment:
You should just be taking the `length` of your plant list and building the message off of that

Reply:
wouldn't i have to create a variable to do that?

Reply:
a local var instead of cluttering up obj/item
ex
```dm
var/list/plants = list()
for (var/obj/item/plant/O in get_turf(src))
	plants += O
message_admins("[length(plants] plants")
```